callmetadata.schema.json

```diff
@@ -5,6 +5,27 @@
     "description": "Meta data supporting dialtone, ringdown, and shoutdown calls",
     "type": "object",
     "definitions": {
+        "state": {
+            "type": "object",
+            "properties": {
+                "state": {
+                    "type": "string",
+                    "description": "Name of the latest state of the current communication: CallOriginated, CallDelivered, CallEstablished, ect. Could be an enum with defined states. ",
+                    "example": "CallDelivered"
+                },
+                "time": {
+                    "type": "string",
+                    "format": "date-time",
+                    "description": "Date and time when the state changed for the given participant",
+                    "example": "2016-06-18T08:05:15Z"
+                },
+                "duration": {
```

Comment:
Looks like this information is redundant since we will have a list of states and the duration is the difference between them. Any issue with that approach?

Reply:
Is there any harm in it being an optional parameter?

Reply:
IMHO It's never a good idea to encourage people to duplicate information, since this can be harmful for data consistency. But, it's allowed to users to add additional properties on definitions so they can add duration(or any other) field on their own responsibility. Additional properties are allowed by default in JSON Schema.

My suggestion is to remove duration from the standard, and if the user needs this information more explicitly they can add as an additional property.